src/components/Dashboard/PrintInfo.js

```diff
@@ -33,9 +21,24 @@ const PrintInfo = ({ isLayout }) => {
                 {`${i18n.t('Print Preview')}: ${infoHeader}`}
             </p>
             <div style={{ maxWidth }}>
-                <p
-                    className={classes.info}
-                >{`${infoPartOne} ${infoPartTwo}`}</p>
+                <p className={classes.info}>
+                    {i18n.t('For best print results:')}
```

Comment:
Can't have a `:` in the translation string bc it is the namespace separator. I guess just put it outside of the `i18n.t`